socorro/external/crashstorage_base.py

```diff
@@ -494,6 +494,10 @@ def __setitem__(self, index, value):
         this method is required by the MutableSequence abstract base class"""
         self.exceptions.__setitem__(index, value)
 
+    def __str__(self):
+        return '%s %s' % (self.message,
+                          ','.join(repr(e[1]) for e in self.exceptions))
```

Comment:
Is it possible for this to kick up a `UnicodeDecodeError`?

Maybe it should be something like this?:

```
def __str__(self):
    return '%s %s' % (self.message,
                      ','.join(repr(e[1]).encode('ascii', 'ignore') for e in self.exceptions))
```